content/post/practical_ml.md

```diff
@@ -83,4 +83,5 @@ If you are lucky enough to work with Data Engineers, then you should certainly t
 
 In order to successfully deploy our model, we must cross our t's and dot our i's. This includes following industry best practices, being scientifically honest in our work, understanding bias in our data and challenging the businesses assumptions throughout. In addition to the generic model building playbook, there are real world considerations that the Data Scientist needs to account for. 
 
-Considerations such as which modeling choices best suit our specific problem, understanding how the business will use our model, and recognizing that our team can add a lot of value. If we orient our work in this way, then we will be able to blend our hard science knowledge with the hard realities of the business world in the most optimal way possible.
+Considerations such as which modeling choices best suit our specific problem, understanding how the business will use our model, and recognizing that our team can add a lot of value. Please see my [follow up post](https://www.locallyoptimistic.com/post/practical_ml_2/) for more tips on what to watch out for. If we orient our work in this way, then we will be able to blend our hard science knowledge with the hard realities of the business world in the most optimal way possible. 
```

Comment:
we should try to get in the habit of using hugo's internal references so that we don't have a ton of broken links if we ever change anything
https://gohugo.io/content-management/cross-references/